docs/fips.md

```diff
@@ -0,0 +1,195 @@
+6     AES256-GCM-SHA384            TLSv1.2                None                None
+7     AES128-SHA                   TLSv1,TLSv1.1,TLSv1.2  None                None
+8     AES256-SHA                   TLSv1,TLSv1.1,TLSv1.2  None                None
+9     ECDHE-RSA-DES-CBC3-SHA       TLSv1,TLSv1.1,TLSv1.2  ECDH,P-521,521bits  prime256v1,secp384r1,secp521r1
+10    DES-CBC3-SHA                 TLSv1,TLSv1.1,TLSv1.2  None                None
+```
+
+### Validating the client configuration
+
+A TLS client sends all of its supported ciphersuites as part of the handshake, in the `ClientHello` message.
+Examining this message requires some additional work, as we need to set up a TLS server with a valid
+certificate that will print it to stdout for us. For the purposes of this document, we're going to use a
+small [Go utility](https://github.com/rgl/tls-dump-clienthello) to do this, but it's also possible with
+OpenSSL's `s_server` command.
+
+```bash
+$ ./tls-dump-clienthello_$(go env GOOS)_$(go env GOARCH)_$(go env GOAMD64)/tls-dump-clienthello
+
+[...]
+
+client version: TLSv1.3
+client version: TLSv1.2
+client cipher suite: TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256 (0xc02b)
+client cipher suite: TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256 (0xc02f)
+client cipher suite: TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384 (0xc02c)
+client cipher suite: TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384 (0xc030)
+client cipher suite: TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305_SHA256 (0xcca9)
+client cipher suite: TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305_SHA256 (0xcca8)
+client cipher suite: TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA (0xc009)
+client cipher suite: TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA (0xc013)
+client cipher suite: TLS_ECDHE_ECDSA_WITH_AES_256_CBC_SHA (0xc00a)
+client cipher suite: TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA (0xc014)
+client cipher suite: TLS_RSA_WITH_AES_128_GCM_SHA256 (0x009c)
+client cipher suite: TLS_RSA_WITH_AES_256_GCM_SHA384 (0x009d)
+client cipher suite: TLS_RSA_WITH_AES_128_CBC_SHA (0x002f)
+client cipher suite: TLS_RSA_WITH_AES_256_CBC_SHA (0x0035)
+client cipher suite: TLS_ECDHE_RSA_WITH_3DES_EDE_CBC_SHA (0xc012)
+client cipher suite: TLS_RSA_WITH_3DES_EDE_CBC_SHA (0x000a)
+client cipher suite: TLS_AES_128_GCM_SHA256 (0x1301)
+client cipher suite: TLS_AES_256_GCM_SHA384 (0x1302)
+client cipher suite: TLS_CHACHA20_POLY1305_SHA256 (0x1303)
+client curve: x25519 (29)
+client curve: secp256r1 (23)
+client curve: secp384r1 (24)
+client curve: secp521r1 (25)
+client point: uncompressed (0)
+
+[...]
+```
+
+## Setting minimum TLS version
+
+While [FIPS 140-3][fips] doesn't, strictly speaking, mandate a specific TLS version, it is nonetheless a good practice to use as recent of a version as possible. Otelcol allows TLS to be configured for each component individually, in a standardized manner. You can find the details [here](https://github.com/open-telemetry/opentelemetry-collector/blob/main/config/configtls/README.md).
```

Comment:
Specifically, versions earlier than 1.2 are considered unsafe. You might find differing opinions in the security/compliance community over whether 1.2 or 1.3 is the best option, currently.